### PR TITLE
feat(cbr): support new parameters

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:
support some parameters:
- vault:
  - backup_name_prefix
- policy:
  - enable_acceleration
  - full_back_interval

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. support new parameters.
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/cbr' TESTARGS='-run=TestAccVault_backupServer'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cbr -v -run=TestAccVault_backupServer -timeout 360m -parallel 4
=== RUN   TestAccVault_backupServer
=== PAUSE TestAccVault_backupServer
=== CONT  TestAccVault_backupServer
--- PASS: TestAccVault_backupServer (382.31s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cbr       382.382s
```
```
make testacc TEST='./huaweicloud/services/acceptance/cbr' TESTARGS='-run=TestAccPolicy_replication'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cbr -v -run=TestAccPolicy_replication -timeout 360m -parallel 4
=== RUN   TestAccPolicy_replication
=== PAUSE TestAccPolicy_replication
=== CONT  TestAccPolicy_replication
--- PASS: TestAccPolicy_replication (7.71s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cbr       7.784s
```

### CODE REVIEW

File: huaweicloud/services/cbr/resource_huaweicloud_cbr_policy.go [hunk 1]
```diff
@@ -144,6 +151,13 @@ func ResourcePolicy() *schema.Resource {
 							Optional:    true,
 							Description: "The latest backup of each year is saved in the long term.",
 						},
+						"full_backup_interval": {
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/cbr/resource_huaweicloud_cbr_policy.go:154]: this parameter not contained in buildPolicyOpDefinition

File: huaweicloud/services/cbr/resource_huaweicloud_cbr_policy.go [hunk 2]
```diff
@@ -101,6 +101,13 @@ func ResourcePolicy() *schema.Resource {
 				Optional:    true,
 				Description: "The name of the replication destination region.",
 			},
+			"enable_acceleration": {
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/cbr/resource_huaweicloud_cbr_policy.go:104]: this parameter not contained in buildPolicyOpDefinition

File: huaweicloud/services/cbr/resource_huaweicloud_cbr_policy.go (hunk 1 above)
Comment (Lance52259) [reply huaweicloud/services/cbr/resource_huaweicloud_cbr_policy.go:154]: fixed, the problem has been occured in the latest rebase option

File: huaweicloud/services/cbr/resource_huaweicloud_cbr_policy.go (hunk 2 above)
Comment (Lance52259) [reply huaweicloud/services/cbr/resource_huaweicloud_cbr_policy.go:104]: fixed, the problem has been occured in the latest rebase option

**Comment by Lance52259** [comment]
```
make testacc TEST='./huaweicloud/services/acceptance/cbr' TESTARGS='-run=TestAccVault_backupServer'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cbr -v -run=TestAccVault_backupServer -timeout 360m -parallel 4
=== RUN   TestAccVault_backupServer
=== PAUSE TestAccVault_backupServer
=== CONT  TestAccVault_backupServer
--- PASS: TestAccVault_backupServer (406.71s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cbr       406.790s
```

**Pull request merged**